Documentation/CodeSnippets/Events/Core/ModifyRecordOverlayIconIdentifierEvent.rst.txt

```diff
@@ -0,0 +1,27 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
```

Comment:
Please regenerate the snippets after https://github.com/TYPO3-Documentation/codesnippet/pull/20 was merged and update your snippets project accordingly.